src/components/data.js

```diff
@@ -0,0 +1,130 @@
+      <>
+        Overall, mathematics and basic competitive programming skills. Please join our <a href="https://discord.gg/fSaFuT3PaA" target="_blank" rel="noopener noreferrer">Discord server</a> for further information about the selection phases.
+      </>
+    )},
+    {question: "I don't have any programming background, can I still join Jollybee?", answer: (
+      <>
+        Yes! You can prepare yourself by learning the basics before the selection phase begins. Check out our <Link to="/learn">Learning Page</Link> for learning recommendations and join our <a href="https://discord.gg/fSaFuT3PaA" target="_blank" rel="noopener noreferrer">Discord server</a> to learn more about how to prepare yourself.
+      </>
+    )},
+    {question: "Do I get SAT points from Jollybee?", answer: "You can get SAT points from participating competitions. Some of Jollybee members even have SAT overflow."},
+    {question: "Do I need to pay for the registration?", answer: "No, it's 100% free!"},
+    {question: "Can I join Jollybee and still participate in other organizations?", answer: "Yes, as long as you have a good time management and willing to commit in participating Jollybee's events."}
+];
+
+// === About.jsx data ===
+
+export const companyListAbout = [
+    {name: "Huawei", link: "https://www.huawei.com/minisite/icpc2024/en/"},
+    {name: "Google", link: "https://codingcompetitionsonair.withgoogle.com/"},
+    {name: "Meta", link: "https://www.facebook.com/codingcompetitions/hacker-cup"},
+    {name: "Shopee", link: "https://careers.shopee.sg/codeleague/"},
+    {name: "Codegoda", link: "https://codegoda.io/"},
+    {name: "Acer", link: "https://news.acer.com/acer-sponsors-the-international-olympiad-in-informatics-2022"},
+    {name: "Traveloka", link: "https://competition.binus.ac.id/icpc2018/"},
+    {name: "Sirclo", link: "http://ngoding-seru.jonathanirvin.gs/"},
+];
+
+export const socialList = [
+    {name: "Instagram", link: "https://www.instagram.com/binusjollybee/"},
+    {name: "Linkedin", link: "https://www.linkedin.com/company/jollybee-binus/"}
+];
+
+// === Benefits.jsx data ===
+export const cpBenefitList = [
+    {icon: `<i class="fa-solid fa-face-smile"></i>`, name: "It's fun!"},
+    {icon: `<i class="fa-solid fa-brain"></i>`, name: "Learn to think efficiently and effectively"},
+    {icon: `<i class="fa-solid fa-puzzle-piece"></i>`, name: "Improve your problem-solving skills"},
+    {icon: `<i class="fa-solid fa-code"></i>`, name: "Improve your general coding skills"},
+    {icon: `<i class="fa-solid fa-circle-nodes"></i>`, name: "Be proficient at algorithms and data structure."},
+    {icon: `<i class="fa-solid fa-user-group"></i>`, name: "Learn to code and solve problems as a team"},
+    {icon: `<i class="fa-solid fa-briefcase"></i>`, name: "Get industry recognition"},
+    {icon: `<i class="fa-solid fa-file"></i>`, name: "Win achievements for your CV"}
+];
```

Comment:
Optional, you can instead put the icon tag inside `cpBenefit` file and save only the class name in the benefits. That way, you can keep consistency of the icon styles.